pkg/controllers/namespace/namespace_controller.go

```diff
@@ -241,10 +241,10 @@ func (c *namespaceController) syncToDatastore(key string) error {
 			return nil
 		}
 
-		// The profile already exists - update it.
-		p.ResourceVersion = gp.ResourceVersion
+		// The profile already exists - update it to have the new spec.
```

Comment:
nit:  should `gp` be `prof` or `pr` or something like that?

Reply:
gp I think stands for `getprofile`, as in the profile we got from the datastore.